siddpubs-conf.bib

```diff
@@ -56,6 +56,13 @@ @string{cvpr
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{kumar2019lego,
+ title={LEGO: Leveraging Experience in Roadmap Generation for Sampling-Based Planning},
+ author={Kumar, R. and Mandalika, A. and Choudhury, S. and Srinivasa, S.},
```

Comment:
```suggestion
 author={Kumar, R. and Mandalika, A. and Choudhury, S. and Srinivasa, S.S.},
```

Reply:
The official submission was made without the middle initial since that is how Sidd is registered on PaperPlaza.

Reply:
Since this is for our records, committing the suggestion.